app.js

```diff
@@ -1,8 +1,140 @@
+// Return the digit in the ones place of an integer
+function getOnesDigit(num) {
+  return num % 10;
+}
+
+// Return the digit in the tens place of an integer
+function getTensDigit(num) {
+  num %= 100;
```

Comment:
That's a cleaner way to do the remainder operations than the way I did. Finding the remainder from the next highest digit place cleans up having to find both whole amounts and remainders ... anyhoo